tests/integration/profile_update/test_daily_profile_monitoring_integration.py

```
@@ -0,0 +1,442 @@
+            break  # async for는 한 번만 실행
+
+    @pytest_asyncio.fixture(scope="function")
+    async def cleanup_test_data(self):
+        """테스트 정리"""
+        yield  # 테스트 실행
+        
+        # 테스트 후 정리
+        async for session in get_async_session():
+            try:
+                # user_id로 프로필들을 찾아서 삭제
+                from sqlalchemy import select, delete
```

Ruff (0.11.9)

`sqlalchemy.delete` imported but unused

Remove unused import: `sqlalchemy.delete`

(F401)